### PR TITLE
feat(kubeconfig): Support renaming the context that is downloaded

### LINKED ISSUE #2: Add ability to rename configs

### PR DESCRIPTION
**Pull request by calebschoepp**
Closes #2

### CODE REVIEW

File: cmd/lkecli/main.go [hunk 1]
```diff
@@ -135,8 +136,15 @@ func (k *kubeconfigCmd) Run(cfg *cfg) error {
 		return err
 	}
 
+	var contextName string
+	if k.Rename != "" {
+		contextName = k.Rename
```
Comment (endocrimes) [inline cmd/lkecli/main.go:141]: We could rename it by default to the cluster name?

Comment (calebschoepp) [reply cmd/lkecli/main.go:141]: Yeah I like that, so still leave `name:"rename" default:""` but than modify that in the code to default to the cluster arg. Or is there a way to do dependent defaults with this cli library?

Comment (endocrimes) [reply cmd/lkecli/main.go:141]: Yeah - so this bit becomes `contextName = (k.Rename || cluster-pretty-name)`

File: cmd/lkecli/main.go [hunk 2]
```diff
@@ -125,7 +126,13 @@ func (k *kubeconfigCmd) Run(cfg *cfg) error {
 		return err
 	}
 
-	newCfg, err := mergeConfigs(k.Output, data)
+	var contextName string
+	if k.Rename != "" {
+		contextName = k.Rename
+	} else {
+		contextName = fmt.Sprintf("%s-ctx", cluster.Label)
+	}
```
Comment (endocrimes) [inline cmd/lkecli/main.go:134]:
```suggestion
	contextName := cluster.Label
	if k.Rename != "" {
		contextName = k.Rename
	}
```

**Review by endocrimes — APPROVED** [review]
LGTM, dropping one tiny change inline to avoid `kubectl config set-context something-ctx` stuttering

**Pull request merged**